resource-watch/resource-watch
# Feat / myrw collections section

## Overview

new collections tab on myrw, makes it possible to edit / create new collections 

![screen shot 2018-03-08 at 17 14 16](https://user-images.githubusercontent.com/971129/37161917-40b3074a-22f4-11e8-935e-a8dc2df00979.png)


## Testing instructions
go to : http://localhost:9000/myrw/collections and make sure you can (create new, edit previous, delete) collection

## Pivotal task
https://www.pivotaltracker.com/story/show/155782343
